lib/membrane_mp4/muxer/cmaf.ex

```diff
@@ -10,12 +10,18 @@ defmodule Membrane.MP4.Muxer.CMAF do
   """
   use Membrane.Filter
```

Comment:
I think the order of callbacks in this module makes it harder to read, requiring many jumps. Trying to follow a single track's "lifetime" I ended up with: `handle_init` -> `handle_pad_added` -> `handle_caps`  -> `handle_demand` -> `handle_process` -> `handle_end_of_stream`.